splunk-otel-android/src/test/java/com/splunk/rum/RumInitializerTest.java

```diff
@@ -90,9 +90,7 @@ SpanExporter buildFilteringExporter(ConnectionUtil connectionUtil) {
         assertTrue(events.size() > 0);
         checkEventExists(events, "connectionUtilInitialized");
         checkEventExists(events, "exporterInitialized");
-        checkEventExists(events, "sessionIdInitialized");
```

Comment:
Hmmm. Did we lose the events `sessionIdInitialized` and `openTelemetrySdkInitialized`? That seems like a potentially breaking change.

Reply:
I need to verify this, but I hope that these are purely informational, just for debugging. I mean, some of these (`connectionUtilInitialized`) are so low-level that the customers shouldn't really care about them.

Reply:
I would just want some assurance that the backend/experience isn't relying on these.

Reply:
We've received confirmation that this is ok.